src/Security/Certificate.cs

```diff
@@ -263,19 +270,47 @@ public NSData GetPublicKey ()
 			}
 		}
 #else
-		[iOS (10,3)]
-		[TV (10,3)]
-		[Deprecated (PlatformName.iOS, 12,0)]
-		[Deprecated (PlatformName.TvOS, 12,0)]
-		[Deprecated (PlatformName.WatchOS, 5,0)]
+#if !NET
+		[iOS (10,3), TV (10,3)]
+#else
+		[SupportedOSPlatform ("ios10.3")]
+		[SupportedOSPlatform ("tvos10.3")]
+#endif
+#if !NET
+		[Deprecated (PlatformName.iOS, 12, 0, message: "This method is no longer available.")]
+		[Deprecated (PlatformName.TvOS, 12, 0, message: "This method is no longer available.")] 
+		[Deprecated (PlatformName.WatchOS, 5, 0, message:  "This method is no longer available.")]
```

Comment:
where does that `"This method is no longer available."` comes from ?

Reply:
I just added these at the beginning since there was no message.. I think I should just not have a message then?

Reply:
The correct message is "Use 'SecCertificateCopyKey' instead."